resqpy/surface/triangulated_patch.py

```diff
@@ -307,6 +298,22 @@ def set_from_sparse_mesh(self, mesh_xyz):
         self.triangles = triangles[:nt, :]
         self.triangle_count = nt
 
+    def get_indices_from_sparse_meshxyz(self, mesh_xyz):
```

Comment:
These new methods can be made private by adding a double underscore before the start of the name.

Reply:
Agree that we should clearly delineate the public and non-public API.

PEP8 says "Use one leading underscore only for non-public methods and instance variables":

https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables